src/plugins/intel_cpu/tests/functional/custom/behavior/ov_executable_network/properties.cpp

```diff
@@ -327,4 +327,18 @@ TEST_F(OVClassConfigTestCPU, smoke_CpuExecNetworkCheckCPUExecutionDevice) {
     ASSERT_EQ(value.as<std::string>(), "CPU");
 }
 
+TEST_F(OVClassConfigTestCPU, smoke_CpuExecNetworkCheckCPURuntimOptions) {
```

Comment:
We also need to add test that check that user properties passes to Core or to compile_model(...) has higher priority than smt specified in model rt_info.

Reply:
Done, thx!